Sources/PostmarkClient/MailClient.swift

```diff
@@ -0,0 +1,74 @@
+import HTTP
+import Mail
+import Vapor
+
+
+public final class MailClient {
+    
+    let serverToken: String
+    let client: ClientProtocol
+    public var result : Response?
+    
+    public init(clientProtocol: ClientProtocol.Type, serverToken: String) throws {
+        self.serverToken = serverToken
+        client = try clientProtocol.make(scheme: "https", host: "api.postmarkapp.com")
+    }
+}
+
+extension EmailAddress {
+    var smtpLongFormatted: String {
+        var formatted = ""
+        
+        if let name = self.name {
+            formatted += name
+            formatted += " "
+        }
+        formatted += "<\(address)>"
+        return formatted
+    }
+}
+
+extension Sequence where Iterator.Element == EmailAddress {
+    var smtpLongFormatted: String {
+        return self.map { $0.smtpLongFormatted } .joined(separator: ", ")
+    }
+}
+
+
+extension MailClient: MailClientProtocol {
+    
+    public convenience init() throws {
+        guard let clientProtocol = Provider.clientProtocol else {
+            throw Provider.Error.noClient
+        }
+        guard let serverToken = Provider.serverToken else {
+            throw Provider.Error.missingConfig("serverToken")
+        }
+        try self.init(clientProtocol: clientProtocol, serverToken: serverToken)
+    }
+    
+    public func send(_ emails: [Mail.Email]) throws {
+        
+        try emails.forEach { email in
+
+            let jsonEmail = JSON([
```

Comment:
I would definitely include a `PostmarkEmail`, somewhat like [the Sendgrid version](https://github.com/bygri/mail/blob/sendgrid/Sources/SendGridClient/Models/SendGridEmail.swift)